### ISSUE TITLE
Increase Accessibility for Keyboard, Screen Reader, and iOS/VoiceOver users

### ISSUE THREAD

**Issue by dovematrix** [flowplayer/flowplayer#1453]
Could you please increase the accessibility of the FlowPlayer?  My university has purchased a commercial license 7.2.4.  Keyboard and screen-reader users on desktop cannot play video since it doesn’t appear to receive any focus. iOS/VoiceOver users also cannot play the video.

**Comment by nnarhinen** [comment]
Hello. Flowplayer7 will not receive any new features anymore. For better accessibility you should take a look at our latest player called "Flowplayer Native". Accessibility documentation for that can be found from https://flowplayer.com/developers/player/accessibility
